pkg/query-service/constants/constants.go

```diff
@@ -55,6 +56,8 @@ var DurationSortFeature = GetOrDefaultEnv("DURATION_SORT_FEATURE", "true")
 
 var TimestampSortFeature = GetOrDefaultEnv("TIMESTAMP_SORT_FEATURE", "true")
 
+var PreferRPMFeature = GetOrDefaultEnv("PREFER_RPM_FEATURE", "false")
```

Comment:
Can you move this to flag in main.go?

Reply:
Why? 
We are using env variables for feature flags.
Eg: https://signoz.io/docs/operate/feature-flags/#adding-configs-to-query-service

Reply:
The feature "flag" usually goes in the CLI flags. This is a pretty common practice, and you can see it used (the closest example of what we rely on is a [collector](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/docker-compose.yaml#L218)) in many executable go applications (some argparse underlying libraries support the env version of the flag with `_`, but it's the CLI flag that makes it possible). We may have used env in the past, but I believe we shouldn't do that going forward for feature flags.

Reply:
It might be a common practice, can you list the reasons behind it? How will it be easier for SigNoz users or us to use CLI flags rather than env variables?
I am not against CLI flags or have a hard opinion but I was thinking that it will bloat the APIhandler code. But that should be a much of a problem.

Reply:
It's easier to discover and manage. When we use the env for both feature flag and general purpose, they are all over the place with mixed usage with no distinction. What does the feature flag do and what type values does it accept? What are the default values and does it return error when incorrect type given or does it silently skip? and answer to these questions is usually the flags. There maybe other reasons which I am not aware.